robot.py

```diff
@@ -152,55 +169,65 @@ def initSwervometer(self, config):
             teamGyroAdjustment = 180 # Blue Team faces 0 degrees at start.
             teamMoveAdjustment = -1 # Blue Team start is oriented 180 degrees from field.
 
-        #self.dashboard.putBoolean(DASH_PREFIX, 'Team is Red', self.team_is_red)
+        return teamGyroAdjustment, teamMoveAdjustment
 
-        self.log("FIELD_START_POSITION:", config['FIELD_START_POSITION'])
+    def updateFieldStartPosition(self, fieldStartPosition):
```

Comment:
You may want to use a different name for this method to avoid confusion with the one in swervometer.